pfinance/functions.py

```diff
@@ -257,6 +257,49 @@ def straight_line_depreciation(purchase_price: float, salvage_value: float, usef
     return (purchase_price - salvage_value) / useful_life
 
 
+def double_declining_balance_depreciation(
+    purchase_price: float,
+    salvage_value: float,
+    useful_life: int,
+    factor: float = 2.0,
```

Comment:
Is there a reason for making a declining balance a positive value? Might be more clear to use 0.5 for the same effect

Reply:
I want to follow the format that Excel uses